client/src/pages/SubmissionView.vue

```diff
@@ -0,0 +1,507 @@
+                  nisi vitae suscipit tellus mauris a. Non nisi est sit amet
+                  facilisis magna. Turpis massa tincidunt dui ut ornare lectus.
+                  Vel risus commodo viverra maecenas accumsan lacus vel
+                  facilisis. Mauris cursus mattis molestie a iaculis at. Aenean
+                  sed adipiscing diam donec. Nisl nunc mi ipsum faucibus. Sed
+                  elementum tempus egestas sed sed risus. Risus pretium quam
+                  vulputate dignissim suspendisse in. Ut faucibus pulvinar
+                  elementum integer. Volutpat ac tincidunt vitae semper quis.
+                  Cras semper auctor neque vitae tempus. Malesuada bibendum arcu
+                  vitae elementum curabitur vitae nunc sed. Tortor vitae purus
+                  faucibus ornare suspendisse sed. Turpis tincidunt id aliquet
+                  risus feugiat. Mauris augue neque gravida in fermentum et
+                  sollicitudin ac orci. In pellentesque massa placerat duis
+                  ultricies lacus sed.
+                </p>
+              </q-card-section>
+              <q-card-actions class="q-pa-md">
+                <q-btn bordered color="primary">Reply</q-btn>
+              </q-card-actions>
+            </q-card>
+            <div class="q-mx-md q-mb-md">
+              <q-separator />
+              <q-card flat square class="bg-grey-1">
+                <q-card-section>
+                  <div class="row items-start no-wrap">
+                    <div class="col">
+                      <div class="text-subtitle1">Egestas</div>
+                      <div>
+                        <small>
+                          <q-icon size="sm" name="subdirectory_arrow_right" />
+                          <span>Reply to Magna Fringilla</span>
+                        </small>
+                      </div>
+                      <small> February 18th, 2021 at 6:35pm</small>
+                    </div>
+
+                    <div class="col-auto">
+                      <q-btn
+                        color="grey-7"
+                        round
+                        flat
+                        icon="more_vert"
+                        aria-label="Reply Comment Menu"
+                      >
+                        <q-menu cover auto-close>
+                          <q-list>
+                            <q-item clickable>
+                              <q-item-section>Remove Card</q-item-section>
+                            </q-item>
+                            <q-item clickable>
+                              <q-item-section>Send Feedback</q-item-section>
+                            </q-item>
+                            <q-item clickable>
+                              <q-item-section>Share</q-item-section>
+                            </q-item>
+                          </q-list>
+                        </q-menu>
+                      </q-btn>
+                    </div>
```

Comment:
Same deal as in InlineComments.  Im not sure if they can share the same component (i.e. will have the same actions), but that could perhaps be a refactor if the two end up having different context menus.

Reply:
This has been refactored as suggested.